### PR TITLE
Fix missed PSR extension for CI workflow

### PR DESCRIPTION
**Pull request by AlexNDRmac**
Hello!

* Type: bug fix
* Link to issue:

In raising this pull request, I confirm the following:

- [X] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [ ] I updated the CHANGELOG

Small description of change:

- fixed missed PSR ext error for Linux CI builds with PHP 7.1-7.2

Thanks

### CODE REVIEW

File: .github/workflows/build-linux.yml [hunk 1]
```diff
@@ -60,6 +55,18 @@ jobs:
...
+        run: |
+          echo "::set-env name=SYMFONY_PHPUNIT_VERSION::${{ matrix.symfony_phpunit }}"
+          echo "::set-env name=ZEPHIR_PARSER_VERSION::v1.3.3"
+          echo "::set-env name=PHP_INI_DIR::$(php-config --configure-options | tr -s " " "\012" | grep with-config-file-scan-dir | cut -d= -f2)"
+
+      - name: Enable PSR extension
+        run: |
+          if ! php --ri psr 1>/dev/null; then
+            echo 'extension="psr.so"' | sudo tee "$PHP_INI_DIR/psr.ini"
+          fi
```
Comment (sergeyklay) [inline .github/workflows/build-linux.yml:68]: @shivammathur Is this expected?

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you

File: .github/workflows/build-linux.yml (hunk 1 above)
Comment (shivammathur) [reply .github/workflows/build-linux.yml:68]: No, this step is not required. `ext-psr` is installed using the package from [`ppa:ondrej/php`](https://launchpad.net/~ondrej/+archive/ubuntu/php) which enables the extension itself upon installation.
Here is a test workflow - https://github.com/shivammathur/test-setup-php/actions/runs/31796827/workflow

Comment (sergeyklay) [reply .github/workflows/build-linux.yml:68]: As you can see in the previous commit of this PR it is not enabled by default, only installed.

Comment (sergeyklay) [reply .github/workflows/build-linux.yml:68]: > Warning: Cannot locate class "Psr\Http\Message\MessageInterface" when extending interface "Test\Integration\Psr\Http\Message

`Psr\Http\Message\MessageInterface` is provided by PSR extension

Comment (sergeyklay) [reply .github/workflows/build-linux.yml:68]: https://github.com/phalcon/zephir/runs/410729035#step:16:217

**Pull request merged**